src/archiefvernietigingscomponent/destruction/models.py

```diff
@@ -294,6 +295,14 @@ class ArchiveConfig(SingletonModel):
             "Only zaken with archiefactiedatum less that this date would be displayed for destruction"
         ),
     )
+    short_review_zaaktypes = ArrayField(
+        models.URLField(_("cases with short review process"), max_length=1000),
```

Comment:
Can you add the translations as well? :)

Reply:
Updated translation files!